stateoftheuniverse/widgets/moon_functions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+"""
+A function to return moon phase & moon brightness.
+"""
+
+import numpy as np
+import astroplan
+import astropy
+from astropy.time import Time
+import matplotlib.pyplot as plt
+from astropy.coordinates import get_moon, get_sun
+import datetime
+import warnings
+warnings.filterwarnings('ignore')
+
+def get_moon_illumination(time: datetime) -> float:
+    """ A function to return the moon illumination
+
+    I'll fill this in later
+    """
+
+    astropytime = Time(time)
+    try:
+        illumination = astroplan.moon.moon_illumination(astropytime)
+    except:
+        print("Something has gone wrong. Please raise an issue on GitHub")
+    return illumination*100
+
+def get_moon_state(time: datetime) -> bool:
+    """ A function to return whether the moon is waxing or waning
+    """
+    astropytime = Time(time)
+    moonra = get_moon(time).ra.value
+    sunra = get_sun(time).ra.value
+    opposite = sunra + 180.
+    if opposite > 360.:
+        opposite -= 360.
+
+    #Moon goes for west to east
```

Comment:
To quote the [PEP8](https://www.python.org/dev/peps/pep-0008/#block-comments), _"Each line of a block comment starts with a # *and a single space* (unless it is indented text inside the comment)."_ ;-)